amd/src/quizgame.js

```diff
@@ -573,7 +579,8 @@ define(['jquery','core/yui', 'core/notification', 'core/ajax'], function($, Y, n
                 currentPointsLeft -= this.fraction;
                 score -= 1000 * this.fraction;
             }
-            if (currentPointsLeft <= 0 && this.level == level && player.alive) {
+            if (--enemyShipCount === 0 && (currentPointsLeft < this.fraction || currentPointsLeft <= 0)
```

Comment:
Really we should call a new function called shipReachedEnd and have that traverse currentTeam to see if any alive.

This still fails if someone shoots a multiple choice only worth 50%

Reply:
There may be something just slightly wrong in the logic here that doesn't get rid of the 100% answer, that won't involve rearchitecting. But my debug environment is down so I can't run through it myself

Reply:
I see what you mean, I'll be reworking this soon now that I'm done with fixes from #49